src/query/interpreter.cpp

```diff
@@ -2799,6 +2816,9 @@ Interpreter::PrepareResult Interpreter::Prepare(const std::string &query_string,
       }
     }
 
+    utils::OnScopeExit active_transaction_update_metric(
+        []() { memgraph::metrics::IncrementCounter(memgraph::metrics::ActiveTransactions); });
```

Comment:
Sorry I probably expressed myself wrongly but I thought under the condition. And you don't need here OnScopeExit, just  put it before dbaccessor is created